diff --git a/CMakeLists.txt b/CMakeLists.txt
index f3afcd4..08ba504 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,25 +46,6 @@ IF( NOT IS_SUNOS )
 
 ENDIF()
 
-IF(EXISTS "${CMAKE_ROOT}/Modules/CPack.cmake")
-	SET(CPACK_BINARY_DEB "ON")
-	SET(CPACK_BINARY_RPM "OFF")
-	SET(CPACK_BINARY_STGZ "OFF")
-	SET(CPACK_BINARY_TBZ2 "OFF")
-	SET(CPACK_BINARY_TGZ "OFF")
-	SET(CPACK_BINARY_TZ "OFF")
-
-	SET(CPACK_PACKAGE_NAME "libsocket")
-	SET(CPACK_PACKAGE_VERSION "2.4.0")
-	SET(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "amd64")
-	SET(CPACK_DEBIAN_PACKAGE_DEPENDS "")
-	SET(CPACK_PACKAGE_CONTACT "Lewin Bormann <lbo@spheniscida.de>")
-	SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "The ultimate socket abstraction library for C and C++.")
-
-	# List executables
-	INCLUDE(CPack)
-ENDIF(EXISTS "${CMAKE_ROOT}/Modules/CPack.cmake")
-
 set(INSTALL_LIB_DIR lib CACHE PATH "Installation directory for libraries")
 set(INSTALL_BIN_DIR bin CACHE PATH "Installation directory for executables")
 set(INSTALL_INCLUDE_DIR include CACHE PATH "Installation directory for header files")
